src/config.prod.json

```diff
@@ -0,0 +1,5 @@
+{
+	"wordpress.com": {
+		"auth": "proxy"
+	}
+}
```

Comment:
Do you think we can merge this like this for now, (I have a build-for-deploy script using this) and we try to replace with oauth2 in a follow up PR?